Rocket.toml

```diff
@@ -6,8 +6,10 @@ key = "key.pem"
 address = "localhost"
 port = 8000
 log = "normal"
+secret_key = "u4shYQ/3Jm6+vh5QZsy4oX84rHDcS3/RDOgUEAL0p4M="
 
 [production]
-address = "0.0.0.0"
+address = "localhost"
```

Comment:
Do not change this. If you need to run dev, then set ROCKET_ENV to "dev" in your environment variables.